packages/trader/src/App/Containers/RealAccountSignup/checkbox.jsx

```diff
@@ -0,0 +1,28 @@
+import classNames from 'classnames';
+import PropTypes  from 'prop-types';
+import React      from 'react';
+
+class Checkbox extends React.PureComponent {
```

Comment:
Isn't there a checkbox in components already?

Reply:
Yes it does. I had a problem with the Checkbox back in the early days of this branch, I had to add this, I will refactor it in the improvement card.